Tests/ApolloCacheDependentTests/WatchQueryTests.swift

```diff
@@ -4,700 +4,986 @@ import ApolloTestSupport
 import StarWarsAPI
 
 class WatchQueryTests: XCTestCase, CacheTesting {
-
+  
   var cacheType: TestCacheProvider.Type {
     InMemoryTestCacheProvider.self
   }
   
-  func testRefetchWatchedQuery() throws {
-    let query = HeroNameQuery()
-
-    let initialRecords: RecordSet = [
-      "QUERY_ROOT": ["hero": Reference(key: "hero")],
-      "hero": [
-        "name": "R2-D2",
-        "__typename": "Droid",
-      ]
-    ]
-
-    withCache(initialRecords: initialRecords) { cache in
-      let store = ApolloStore(cache: cache)
-      let networkTransport = MockNetworkTransport(body: [
-        "data": [
-          "hero": [
-            "name": "Artoo",
-            "__typename": "Droid"
+  var cache: NormalizedCache!
+  var server: MockGraphQLServer!
+  var client: ApolloClient!
+  
+  override func setUpWithError() throws {
```

Comment:
No, just that they are currently defined on `XCTestCase` so the super implementations don't do anything. But you're right, it's more robust to have them call `super` in case we ever want to have a superclass with common initialization.